R/session_renew.R

```diff
@@ -55,3 +55,31 @@ session_renew = function(password, ..., username, session=current_session()){
 		invisible(authenticate_sb(sb_username, password))
 	}
 }
+
+
+refresh_token_before_expired <- function(refresh_amount_seconds = 600) {
```

Comment:
How long do the tokens generally last?

Reply:
I don't know -- this is the constant from sciencebasepy though.